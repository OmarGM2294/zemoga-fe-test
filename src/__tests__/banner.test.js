import React from 'react'

import { create } from 'react-test-renderer'

import Banner from '../components/banner'

describe('Banner', () => {
  it('Should match snapshot (it renders correctly)', () => {
    const component = create(<Banner />)

    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })
})
