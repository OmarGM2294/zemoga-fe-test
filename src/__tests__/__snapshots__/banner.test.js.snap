// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner Should match snapshot 1`] = `
<aside
  aria-label="Speak Out"
  role="doc-tip"
>
  <div>
    <span>
      Speak out. Be heard.
    </span>
    <span>
      Be counted
    </span>
  </div>
  <div>
    <p>
      Rule of Thumb is a crowd sourced court of public opinion where anyone and everyone can speak out and speak freely. Itâ€™s easy: You share your opinion, we analyze and put the data in a public report.
    </p>
  </div>
  <button
    aria-label="close"
  >
    <div
      style={
        Object {
          "bottom": 0,
          "boxSizing": "border-box",
          "display": "block",
          "left": 0,
          "margin": 0,
          "overflow": "hidden",
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    >
      <img
        decoding="async"
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style={
          Object {
            "border": "none",
            "bottom": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": 0,
            "left": 0,
            "margin": "auto",
            "maxHeight": "100%",
            "maxWidth": "100%",
            "minHeight": "100%",
            "minWidth": "100%",
            "objectFit": undefined,
            "objectPosition": undefined,
            "padding": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "visibility": "hidden",
            "width": 0,
          }
        }
      />
    </div>
  </button>
</aside>
`;
